static/app/views/detectors/detectorListRow.tsx

```diff
@@ -0,0 +1,150 @@
+import styled from '@emotion/styled';
+
+import Checkbox from 'sentry/components/checkbox';
+import {Flex} from 'sentry/components/container/flex';
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import {
+  ConnectionCell,
+  type Item,
+} from 'sentry/components/workflowEngine/gridCell/connectionCell';
+import {EmptyCell} from 'sentry/components/workflowEngine/gridCell/emptyCell';
+import {IssueCell} from 'sentry/components/workflowEngine/gridCell/issueCell';
+import {NumberCell} from 'sentry/components/workflowEngine/gridCell/numberCell';
+import {TitleCell} from 'sentry/components/workflowEngine/gridCell/titleCell';
+import {tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Group} from 'sentry/types/group';
+import type {AvatarProject} from 'sentry/types/project';
+
+export type Detector = {
+  automations: Item[];
+  groups: Group[];
+  id: string;
+  link: string;
+  name: string;
+  project: AvatarProject;
+  details?: string[];
+  disabled?: boolean;
+};
+
+type DetectorListRowProps = Detector & {
+  handleSelect: (id: string, checked: boolean) => void;
+  selected: boolean;
+};
+
+export function DetectorListRow({
+  automations,
+  groups,
+  id,
+  link,
+  name,
+  project,
+  details,
+  handleSelect,
+  selected,
+  disabled = false,
+}: DetectorListRowProps) {
+  return (
+    <RowWrapper disabled={disabled}>
+      <InteractionStateLayer />
```

Comment:
not sure why this is applying hover styling to the entire panel instead of just the individual row 🫠

Reply:
`InteractionStateLayer` uses absolute positioning, so throwing a `position: relative` on the parent element should do the trick.